### PR TITLE
Use llvm 11 on Fedora 33.

### PR DESCRIPTION
**Pull request by toregge**
@baldersheim : please review

### CODE REVIEW

File: eval/src/vespa/eval/eval/llvm/llvm_wrapper.cpp [hunk 1]
```diff
@@ -200,7 +200,12 @@ struct FunctionBuilder : public NodeVisitor, public NodeTraverser {
         }
         assert(pass_params == PassParams::LAZY);
         assert(params.size() == 2);
+#if LLVM_VERSION_MAJOR >= 11
+        return builder.CreateCall(llvm::cast<llvm::FunctionType>(params[0]->getType()->getPointerElementType()),
+                                  params[0], {params[1], builder.getInt64(idx)}, "resolve_param");
+#else
```
Comment (baldersheim) [inline eval/src/vespa/eval/eval/llvm/llvm_wrapper.cpp:206]: When was this signature available ? In 11 or in an earlier version ?

**Review by baldersheim — APPROVED** [review]

File: eval/src/vespa/eval/eval/llvm/llvm_wrapper.cpp (hunk 1 above)
Comment (toregge) [reply eval/src/vespa/eval/eval/llvm/llvm_wrapper.cpp:206]: 7 or earlier. The other signature was deprecated in llvm 8 and removed in llvm 11.

Comment (baldersheim) [reply eval/src/vespa/eval/eval/llvm/llvm_wrapper.cpp:206]: Then I suggest you change to test for the version where the new signature was available. What is the oldest llvm version we require ?

Comment (toregge) [reply eval/src/vespa/eval/eval/llvm/llvm_wrapper.cpp:206]: 7

Comment (baldersheim) [reply eval/src/vespa/eval/eval/llvm/llvm_wrapper.cpp:206]: But then we can just avoid the ifdef and use the new signature directly.

**Pull request merged**